etc/profile-m-z/otter-browser.profile

```diff
@@ -0,0 +1,57 @@
+# Firejail profile for otter-browser
+# Description: Lightweight web browser based on Qt5 
+# This file is overwritten after every install/update
+# Persistent local customizations
+include otter-browser.local
+# Persistent global definitions
+include globals.local
+
+?BROWSER_ALLOW_DRM: ignore noexec ${HOME}
+
+noblacklist ${HOME}/.pki
+noblacklist ${HOME}/.local/share/pki
+noblacklist ${HOME}/.cache/Otter
+noblacklist ${HOME}/.config/otter
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+
+mkdir ${HOME}/.pki
+mkdir ${HOME}/.cache/Otter
+mkdir ${HOME}/.config/otter
+mkdir ${HOME}/.local/share/pki
+whitelist ${DOWNLOADS}
+whitelist ${HOME}/.pki
+whitelist ${HOME}/.cache/Otter
+whitelist ${HOME}/.config/otter
+whitelist ${HOME}/.local/share/pki
+whitelist /usr/share/otter-browser
+include whitelist-common.inc
+include whitelist-runuser-common.inc 
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+netfilter
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+?BROWSER_DISABLE_U2F: nou2f
+protocol unix,inet,inet6,netlink
+seccomp !chroot
+shell none
+
+disable-mnt
+private-cache
```

Comment:
is a private-bin possible?

Reply:
`private-bin` will break right-clicking Downloads and opening files among other things. Opera,Falkon,Firefox Chromium-common, all have none or commented. Do you want me to add the basic stuff like `otter,sh,bash,which`?

Reply:
> will break right-clicking Downloads and opening files

I don't think we support this for other browsers

> Opera,Falkon,Firefox Chromium-common

The reason is that the most distros ship custom/own shell-scripts to start firefox.

Reply:
I'll add the above `private-bin`.